spec/unit/sensu_ad_auth_spec.rb

```diff
@@ -0,0 +1,322 @@
+require 'spec_helper'
+require 'puppet/type/sensu_ad_auth'
+
+describe Puppet::Type.type(:sensu_ad_auth) do
+  let(:default_config) do
+    {
+      name: 'ad',
+      servers: [
+        {'host' => 'test', 'port' => 389},
+      ],
+      server_binding: {'test' => {'user_dn' => 'cn=foo','password' => 'foo'}},
+      server_group_search: {'test' => {'base_dn' => 'ou=Groups'}},
+      server_user_search: {'test' => {'base_dn' => 'ou=People'}},
+    }
+  end
+  let(:config) do
+    default_config
+  end
+  let(:auth) do
+    described_class.new(config)
+  end
+
+  it 'should add to catalog with raising an error' do
```

Comment:
it should add to the catalog with raising an error and then expect to not raise an error? huh?

Reply:
Really unfortunate typo,  `s/with/without/`.  Fixing, but I think this may be present in all the unit tests as I usually copy existing tests for new types.  I added commit to fix other tests where this is also present.